communication/best-practices.md

```diff
@@ -0,0 +1,82 @@
+# Mailing list and Github usage best practice                                                       
+                                                                                
+The Kubernetes Mailing list or Google Groups functions as the primary means of  
+asynchronous communication for the project's                                    
+[Special Interest Groups (SIG)][sig-list] and [Working Groups (WG)][sig-list].  
+That's why you may want to set your filters in your email account to attain a
+good signal-to-noise ratio with regards to the mailing list messages and Github
+notifications. All the steps below are basically for Gmail users.
+
+Note: If you are looking to create and manage lists as a chair or other organizer, 
+check this mailing list [guideline][mailinglist-guidelines] doc.
+
+- [Example for setting filters to Kubernets Mailing lists](#example-for-setting-filters-to-kubernets-mailing-lists)
+- [Examples for setting filters to Kubernetes Github notifications](#examples-for-setting-filters-to-kubernetes-github-notifications)
+                                                                                
+## Example for setting filters to Kubernetes Mailing lists           
+
+It depends on the SIG or/and WG you are involved in. 
+You would like to setup your filters in your Gmail account, so you don't need to have 
+all emails structured in one place.
```

Comment:
I'm confused by what "you don't need to have all emails structured in one place" means? :thinking:

Reply:
maybe, we can say: "you will not have all the incoming emails in one place"?